lib/spire/order.rb

```diff
@@ -149,13 +157,16 @@ def create(options)
         client.create(
           :order,
           'customer' => options[:customer],
+          'status' => options[:status],
           'address' => options[:address],
           'shippingAddress' => options[:shipping_address],
           'items' => options[:items],
           'discount' => options[:discount],
           'freight' => options[:freight],
           'customerPO' => options[:customer_po],
-          'type' => options[:type]
+          'type' => options[:type],
+          'contact' => optionsp[:contact],
```

Comment:
@Dujota typo on 'options'

Reply:
good eye, fixed and i also updated the version #